libr/include/r_flag.h

```diff
@@ -103,6 +103,7 @@ R_API void r_flag_list(RFlag *f, int rad, const char *pfx);
 R_API bool r_flag_exist_at(RFlag *f, const char *flag_prefix, ut16 fp_size, ut64 off);
 R_API RFlagItem *r_flag_get(RFlag *f, const char *name);
 R_API RFlagItem *r_flag_get_i(RFlag *f, ut64 off);
+R_API RFlagItem *r_flag_get_by_spaces(RFlag *f, ut64 off, int nspaces, ...);
```

Comment:
we could remove `nspaces` by forcing the user to add a `NULL` at the end of the spaces, what do you think?